base/strings/util.jl

```diff
@@ -464,11 +464,89 @@ function hex2bytes(s::AbstractString)
 end
 
 """
+    hex2bytes(s::AbstractVector{UInt8})
+
+Convert the hexadecimal bytes array to its binary representation. Returns
+`Vector{UInt8}`, i.e. a vector of bytes.
```

Comment:
This is pretty unclear on what a "hexadecimal bytes vector" and "its binary representation" is.  I would say something like:
```
Given an array `s` of ASCII codes for a sequence of hexadecimal digits, returns a `Vector{UInt8}` of bytes corresponding to the binary representation: each successive pair of hexadecimal digits in `s` gives the
value of one byte in the return vector.   (The length of `s` must be even, and the returned array has
half of the length of `s`.)
```